Gemfile.lock

```diff
@@ -332,9 +336,10 @@ GEM
       rouge (~> 2.0.7)
     xcpretty-travis-formatter (1.0.1)
       xcpretty (~> 0.2, >= 0.0.7)
-    zeitwerk (2.6.0)
+    zeitwerk (2.6.6)
 
 PLATFORMS
+  arm64-darwin
```

Comment:
I will just remove this for now, because it adds `sqlite3 (1.6.0-arm64-darwin)` which makes CI fail